src/main/java/computer_science_basics/homework_1/Task1

```diff
@@ -1,12 +1,10 @@
 Task 1
-* Convert number 456 from decimal to binary = 123123
-* Convert number 01001001011 from binary to decimal = 123123
-* Convert number 450 from octal to decimal = 123123
-* Convert number 345 from decimal to octal = 213123
-* Convert number 542 from decimal to hexadecimal = 123123
-* Convert number 34F1E from hexadecimal to decimal =123123
+* Convert number 456 from decimal to binary = 111001000
+  * Convert number 01001001011 from binary to decimal = 587
+  * Convert number 450 from octal to decimal = 296
+  * Convert number 345 from decimal to octal = 531
+  * Convert number 542 from decimal to hexadecimal = 21E
+  * Convert number 34F1E from hexadecimal to decimal =216862
 
```

Comment:
Please revert this changes. 
You don't need to modify Task1 file
You have already created a new file